--- tasklite-core/app/Main.hs
+++ tasklite-core/app/Main.hs
@@ -204,14 +204,7 @@ nameToAliasList = (
 
 {- Imitates output from `git describe` -}
 versionSlug :: Text
-versionSlug =
-  let
-    gitInfo = $$tGitInfoCwd
-  in
-    fromString $
-      (showVersion version)
-      <> "+" <> take 8 (giHash gitInfo)
-      <> (if giDirty gitInfo then "-dirty" else "")
+versionSlug = "nix"
 
 
 aliasWarning :: Text -> Doc AnsiStyle
